src/sentry/tasks/update_user_reports.py

```diff
@@ -63,7 +65,12 @@ def update_user_reports(**kwargs: Any) -> None:
         for event in events:
             report = report_by_event.get(event.event_id)
             if report:
-                reports_with_event += 1
+                if features.has(
+                    "organizations:user-feedback-ingest", project.organization, actor=None
+                ):
+                    shim_to_feedback(
```

Comment:
When is this task ran? Won't this cause duplicate feedbacks with the call to shim_to_feedback in `userreport.py`?

Reply:
if you look at the above:

```
user_reports = UserReport.objects.filter(
        group_id__isnull=True,
        environment_id__isnull=True,
        date_added__gte=start,
        date_added__lte=end,
    )
```

this is the key -- we're only going to call shim_to_feedback on user reports which haven't already been shimmed, as there was no event associated with them, so no duplication.

Reply:
could we just use `event_id__isnull` instead?

Reply:
don't think it really matters too much -- they should all return equivalent results unless i'm missing something

Reply:
from a reader's perspective, `event_id__isnull` makes it clear that we're filtering on user reports with missing events. Although environment and group = null is correlated with event = null right now, the relationship isn't clear and it might not always be the case

Reply:
@aliu3ntry i think that is a fair point -- my only counter is that this PR is not changing this line, and i'd rather not introduce changes that aren't needed. i could add a comment explaining if that makes sense to you?

Reply:
I see, that's fair. Yes a comment would be good

Reply:
right, totally that makes sense -- the event_id is added regardless if the event exists or not.